app/scripts/services/navigate.js

```diff
@@ -258,6 +258,11 @@ angular.module("openshiftConsole")
               group: rgv.group
             });
         }
+
+        if (_.get(opts, "tab")) {
+          url.removeSearch("tab").addSearch("tab", opts.tab);
```

Comment:
It doesn't appear to. That's what it initially appeared like on testing. Appears as if you have to be careful when testing it in the console since it doesn't give the appearance of updating the returned object used by URI js, that is until it's expanded. Based on that understanding went back through and did a few tests involving...
- replacing an existing "tab" query 
- adding a "tab" parameter in addition to an existing querystring
- and adding, removing, then adding the "tab" query back

they came back as expected.